apis/apis.md

```diff
@@ -5,12 +5,14 @@ title: APIs
 description: Machine translation APIs
 ---
 
-There is a growing set of machine translation engines available.
+There is a growing set of machine translation APIs available.
 
-The most popular engines are available as self-serve APIs and support more than 100 [languages](/languages/languages.md) and more than 10000 language pairs.
+The most popular APIs are available as self-serve APIs and support more than 100 [languages](/languages/languages.md) and more than 10000 language pairs.  By 2022, there was a self-serve API that supported more than 300 languages.
```

Comment:
In our [style guide](https://machinetranslate.org/style#spacing), we established that sentences should be separated by a new line. I know the result is the same, but we should follow the style guide to unify.

Reply:
Should we link to Niutrans when mentioning this fact?
Although this can change in the future, by 2022 it was true.

I'm afraid that, if another API supports more languages in the future, this sentence would end up being misleading as the audience may think the current API that supports many languages is the one we are talking about.

Reply:
I updated the style guide.

Reply:
I'm not too worried about the 300 languages not being evergreen, because it is a rare and notable event, and we can even make it dynamic before that happens.

I believe it will be something like this:
```
{{ site.data.apis | map: "languages" | map: "size" | sort | last }}
```
except that for some APIs we use language pairs.